htdocs/admin/content/galaxy/exploration.inc.php

```diff
@@ -30,37 +42,32 @@
     $radius = 1;
 
     // Discover selected cell
-    if (isset($_POST['discover_selected'])) {
-        $sx = intval($_POST['sx']);
-        $sy = intval($_POST['sy']);
-        $cx = intval($_POST['cx']);
-        $cy = intval($_POST['cy']);
-        $radius = abs(intval($_POST['radius']));
-
-        /** @var CellRepository $cellRepository */
-        $cellRepository = $app[CellRepository::class];
-        /** @var ConfigurationService $config */
-        $config = $app[ConfigurationService::class];
+    if ($request->request->has('discover_selected')) {
+        $sx = $request->request->getInt('sx');
+        $sy = $request->request->getInt('sy');
+        $cx = $request->request->getInt('cx');
+        $cy = $request->request->getInt('cy');
+        $radius = abs($request->request->getInt('radius'));
 
         $cell = $cellRepository->getCellIdByCoordinates($sx, $sy, $cx, $cy);
         if ($cell !== null) {
             [$absX, $absY] = $cell->getAbsoluteCoordinates($config->param1Int('num_of_cells'), $config->param2Int('num_of_cells'));
-            $user->setDiscovered($absX, $absY, $radius);
+            $userUniverseDiscoveryService->setDiscovered($uid, $absX, $absY, $radius);
```

Comment:
Might almost be easier to pass in `$cell` then we don't need the config service here

Reply:
the "cell" does not know its absolute coordinates, as this depends on configuration....

Reply:
done 
eaf3ebd